### DISCUSSION TITLE
How to align the tab position with nvimtree separator?

### DISCUSSION THREAD

**Discussion by MKrbm** [NvChad/NvChad#2329] [Q&A]
Hi there! I have a question about UI.

![image](https://github.com/NvChad/NvChad/assets/57752203/6a075356-8151-443b-a644-e21d4753030b)

As picture shows, the rightmost side of tab is not aligned to separator of nvimtree. 

How can I fix this? 

I'd like to do either of them
- Algin as precise as possible to separator
- Ignore nvimtree and span all the way up to left side of window

Thank you

**Comment by siduck** [comment]
@MKrbm  https://github.com/NvChad/ui/tree/tabufline can you try this branch? go to default plugins list and add "tabufline" branch to the ui plugin over there

**Reply by MKrbm** [reply]
Thank you! 

Now, it sort of fixed, but I'd like to align perfectly with nvim-tree's window separator.

![image](https://github.com/NvChad/NvChad/assets/57752203/4a7852b0-4c45-49e7-be81-eeb43eca1647)

To tell the truth, I looked for some discussions regarding this topic (i.e. setting `vim.opt.fillchars:append("vert:▕")`), but I couldn't be satisfied with them. 
It would be great if you could help me with this. Thank you.


---
Below is a reference.
set `vim.opt.fillchars:append("vert:▕")` but line is too thin.
![image](https://github.com/NvChad/NvChad/assets/57752203/278ebc36-ecaf-434a-b3cb-558c86f0591f)

**Reply by siduck** [reply]
@MKrbm  your solution will do  nothing btw, its just that tabufline sometimes at some screen sizes will show that weird space, i'll try to fix it this weekend

**Reply by MKrbm** [reply]
Thank you very much!
